src/backups/backup.ts

```diff
@@ -427,11 +427,12 @@ export class Backup implements Responsable {
     protected async exportDataDirectories(): Promise<void> {
         const config = app().config;
 
-        let item_list: Array<string> = ['pki', 'policy', 'snapshot'];
+        let item_list: Array<string> = ['openvpn.history', 'pki', 'policy', 'snapshot'];
 
         for (let item of item_list) {
-            const dst_dir = path.join(this._backupPath, Backup.DATA_DIRNAME, item);
+            let dst_dir = path.join(this._backupPath, Backup.DATA_DIRNAME, item);
             if (await FSHelper.directoryExists(config.get(item).data_dir)) {
+                if(item === 'openvpn.history') dst_dir = path.join(this._backupPath, Backup.DATA_DIRNAME, 'archive');
```

Comment:
Why don't set `archive` in the `item_list` array and keep the same logic for the `for` body?

Reply:
I decided not to change the config.js structure, in this case it is openvpn.history. But the path structure had to be DATA/archive/openvpn/history. That's why I do that check.